apps/web/pages/drives/[id].tsx

```diff
@@ -36,7 +36,7 @@ const DrivePage: React.FC<Props> = ({ data }) => {
 					<FirebaseProvider data={data} fullPath={decodeURIComponent(folderPath)}>
 						<Dashboard />
 					</FirebaseProvider>
-				) : data.type === "s3" || data.type === "backblaze" ? (
```

Comment:
We can go with this method for searching: `["s3", "backblaze", "wasabi", "digitalocean"].includes(data.type)`

Reply:
good solution!